run.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+import json
+import os
+import time
+
+import docker
```

Comment:
This requires installing Docker SDK for Python:
```
pip install docker
```

Reply:
Thanks for the catch! Added that to the README